nengo_viz/static/viz.js

```diff
@@ -4,10 +4,21 @@ var VIZ = {};
 VIZ.max_zindex = 0;
 
 /**
+ * Helper function to clip a number, keeping it between two values.
+ */
+VIZ.clip = function(x, low, high) {
+    if (x < low)
```

Comment:
I guess if you're used to Python. In C, I'll sometimes even put this all on one line `if (x < low) x = low;`. It might actually be better as one line (than what it currently is), since then you're not tempted to add another line. But having the curly braces is certainly the safest.